README.md

```diff
@@ -286,13 +279,14 @@ If an adapter implements specific methods you wish to call, then you can use `in
 If your app implements dynamic API keys for various analytics integration, you can defer the initialization of the adapters. Instead of configuring `ember-metrics` through `config/environment`, you can call the following from any Object registered in the container:
 
 ```js
-import Ember from 'ember';
+import { Route } from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
-export default Ember.Route.extend({
-  metrics: Ember.inject.service(),
+export default Route.extend({
+  metrics: service(),
   afterModel(model) {
-    const metrics = Ember.get(this, 'metrics');
-    const id = Ember.get(model, 'googleAnalyticsKey');
```

Comment:
What's the grounds for changing from const to let here?

Reply:
Had the feeling that using `const` only on the top of a file is an established pattern in ember ecosystem but could revert if you prefer.

Reply:
> Had the feeling that using `const` only on the top of a file is an established pattern in ember ecosystem but could revert if you prefer.

I'd try to leave const/let alone, for the same reason that I'd avoid changing tabs to spaces or vice versa

Reply:
Reverted the change.

In general I would recommend to follow the patterns enforced by default ember blueprints (indentation by two spaces) and used by ember guides (prefer `let` over `const`). But I don't care that much to be honest.